### PR TITLE
Create frattura-ed-integrita-strutturale-fracture-and-structural-inte…

### PR DESCRIPTION
**Pull request by POBrien333**
…grity.csl

via https://forums.zotero.org/discussion/94204/style-request-fracture-and-stuctural-integrity-frattura-ed-integrita-strutturale#latest

I used a different style as a base. That linked style was a mess...

### CODE REVIEW

File: frattura-ed-integrita-strutturale-fracture-and-structural-integrity.csl [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    <layout delimiter="," prefix="[" suffix="]">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  <bibliography entry-spacing="0" second-field-align="flush">
+    <sort>
+      <key macro="author"/>
+    </sort>
+    <layout suffix=".">
+      <text variable="citation-number" prefix="[" suffix="]"/>
```
Comment (bwiernik) [inline frattura-ed-integrita-strutturale-fracture-and-structural-integrity.csl:107]: I think This needs a delimiter or suffix with white space.

Comment (POBrien333) [reply frattura-ed-integrita-strutturale-fracture-and-structural-integrity.csl:107]: Is that necessary for `second-field-align="flush"` ?

Comment (adam3smith) [reply frattura-ed-integrita-strutturale-fracture-and-structural-integrity.csl:107]: no, just looks wrong in the auto preview

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**